src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -1646,6 +1646,29 @@ void M()
             }
         }
 
+        [Fact]
+        public void CS0221ERR_ConstOutOfRangeCheckedNullableInt()
```

Comment:
@jcouv Should I remove this test? Exist another test in ```NullableConversionTests`` and that  test is more complete

Reply:
It's fine either way.